tests/server_send.cpp

```diff
@@ -25,6 +25,8 @@
 using namespace ioremap::elliptics;
 using namespace boost::unit_test;
 
+#define ENABLE_LONG_TESTS 0
```

Comment:
Sounds good to me. This option can also be used for other *long* tests when they will appear